package.json

```
@@ -34,7 +34,7 @@
   "dependencies": {
     "@edx/brand": "npm:@edx/brand-openedx@1.2.0",
     "@edx/frontend-component-cookie-policy-banner": "2.2.2",
-    "@edx/frontend-platform": "3.6.1",
+    "@edx/frontend-platform": "^4.0.2",
```

Why restrict it to v4 only?
```suggestion
    "@edx/frontend-platform": "4.0.2",
```

---

Because we released the node18 upgrade in frontend-platform version 4.

---

Adding the caret (^) will update to all future minor/patch versions, without incrementing the major version. We should not restrict it unless we have reason to do it.

---

removed